spec/models/course/assessment/answer/scribing_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Course::Assessment::Answer::Scribing, type: :model do
+  it { is_expected.to act_as(Course::Assessment::Answer) }
+
+  it 'has many scribbles' do
+    expect(subject).to have_many(:scribbles).
+      class_name(Course::Assessment::Answer::ScribingScribble.name).
+      dependent(:destroy)
+  end
+  it { is_expected.to accept_nested_attributes_for(:scribbles).allow_destroy(true) }
+
+  let(:instance) { Instance.default }
+  with_tenant(:instance) do
+    describe '#reset_answer' do
+      let(:answer) {
```

Comment:
Can put this in 1 line. If you wish to define multi-line blocks, use `do ... end`.